setup.py

```diff
@@ -55,7 +55,7 @@ def get_tests_require() -> List[str]:
 def get_extras_require() -> Dict[str, List[str]]:
 
     requirements = {
-        "checking": ["black", "hacking", "isort", "mypy==0.782", "blackdoc"],
+        "checking": ["black", "hacking", "isort", "mypy", "blackdoc"],
```

Comment:
`mypy` at `tox.ini` would also need to be unpinned.

Reply:
You're right. Thank you. dab902e